### ISSUE TITLE
Welcome back! Do you have updates  for the espruino repository?

### ISSUE THREAD

**Issue by tyrone1984** [jeffmer/P8Apps#11]
Welcome back @jeffmer!!
Do you have updates (2v10) for the repository, it is a little older https://github.com/jeffmer/Espruino

**Comment by jeffmer** [comment]
Yes, the updates to the jeffmer/Espruino repository are in the branch JEFFMER. Also, beware, I think there may be a bug to do with NEGATED pins. I had to change the boot and launch P8 apps in this repository as the button, power and buzzer pins were inverted from the previous build.

**Comment by tyrone1984** [comment]
Thanks for reminding,  I will check it.
